outputs.tf

```diff
@@ -1,24 +1,28 @@
 output "truefoundry_db_name" {
-  value = var.create_db == true ? module.postgresql-db[0].server_name : "dummy"
+  value = var.create_db == true ? var.database_name : "dummy"
```

Comment:
In my view it is better to get the outputs from the module outputs instead of reusing input variables as much as possible. This helps in cases when we have to change the input variable names or reuse the same name for something else. Keeps the input and output nicely segregated

Reply:
Agreed, it is better to use the module and don't forward input variables. What was the need for this change?